accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -501,13 +505,10 @@ non-nullable type and no initializer expression, and the class has a generative
 where the variable is not initialized via an initializing formal or an initializer list entry,
 unless the variable is marked with the `late` modifier.
 
-It is an error if a mixin declaration or a class declaration with no generative constructors
-declares an instance variable with a potentially non-nullable type and no initializer expression
-unless the variable is marked with the `late` modifier.
-
-It is an error to derive a mixin from a class declaration which contains
-an instance variable with a potentially non-nullable type and no initializer expression
-unless the variable is marked with the `late` modifier.
+It is an error if a mixin declaration or a class declaration with no generative
```

Comment:
I think we need to make an exception for `abstract` and `external` fields as well, right?  Or we can specify that as part of the abstract fields proposal/spec, but either way we should make sure it's covered (and tested).

Reply:
Yes, and that fits perfectly into the null-safety specification, because that feature is bundled as part of null safety.

It seems funny that we mention that there is no error when the modifier `abstract` or `external` is specified in a non-local variable declaration, because it is a syntax error to _have_ it. But the wording is still necessary because we would otherwise say that _every_ program with a final abstract or external variable is an error because there is no initialization of it anywhere.

Done.